CMakeLists.txt

```diff
@@ -67,6 +74,8 @@ target_include_directories(
 )
 
 set(HEADER_BASE "${CMAKE_CURRENT_SOURCE_DIR}/include")
+target_link_libraries(spdlog INTERFACE fmt::fmt-header-only)
```

Comment:
Why `fmt::fmt-header-only`? I think it should be `fmt::fmt`.

Reply:
> Just another reason why bundled sources are absolute evil :)

I see. Thanks for the lesson :)

It's fixed and rebased.